src/entity/Unit.ts

```diff
@@ -1444,6 +1450,91 @@ export function findLOSLocation(unit: IUnit, target: Vec2, underworld: Underworl
 
 }
 
+
+// handles drawing attack range, bloat radius, and similar graphics each frame while a unit is selected
+export async function drawSelectedGraphics(unit: IUnit, prediction: boolean = false, underworld: Underworld): Promise<void> {
+
+  if (!prediction) {
```

Comment:
Let's have this exclude globalThis.headless too.  It shouldn't cause any errors but there's no reason for the headless server to have to take any of the branches in this function since it's visual only.